sample-queries/sample-queries.json

```diff
@@ -1845,6 +1845,204 @@
+    },
+    {
+      "id": "9ab2d40f-6e69-4f47-a1e7-4d6da2d311a2",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get the passive reverse dns records for a host",
+      "requestUrl": "/beta/security/threatIntelligence/hosts/{hostId}/passiveDnsReverse",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "dcdde342-dc4d-41d1-aebe-e14e7607b541",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get a host component",
+      "requestUrl": "/beta/security/threatIntelligence/hostComponents/{hostComponentId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "c1107e51-c6d5-4833-a316-3433e427eb4f",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get a host cookie",
+      "requestUrl": "/beta/security/threatIntelligence/hostCookies/{hostCookieId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "02ab3782-e837-41f0-8bfe-4d6edea33e35",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get a host tracker",
+      "requestUrl": "/beta/security/threatIntelligence/hostTrackers/{hostTrackerId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "de920480-949b-46fe-b5d4-3179837a0ccb",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get intelligence profiles",
+      "requestUrl": "/beta/security/threatIntelligence/intelProfiles",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "9cd1c98c-4fd0-424d-9197-6d3b092c2f5d",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get an intelligence profile",
+      "requestUrl": "/beta/security/threatIntelligence/intelProfiles/{intelligenceProfileId}",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "864d61af-7bb2-4f10-8bcd-8b8c2d887957",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get indicators for an intelligence profile",
+      "requestUrl": "/beta/security/threatIntelligence/intelProfiles/{intelligenceProfileId}/indicators",
+      "docLink": "https://github.com/microsoftgraph/microsoft-graph-docs/pull/20472/files?short_path=bbfa479",
+      "skipTest": false
+    },
+    {
+      "id": "1a52c363-bfb0-40c7-9502-014918989611",
+      "category": "Security",
+      "method": "GET",
+      "humanName": "get intelligence profile indicator",
```

Comment:
Maybe change this to `get an intelligence profile indicator` to match other request for an individual entity?